### ISSUE TITLE
Problem with select2 multiple-select on Safari

### ISSUE THREAD

**Issue by tmconnect** [wp-shortcake/shortcake#753]
On Safari (10.1.1) the post_select field doesn't work correctly. Entering the field drops down the notice "Please enter 1 or more characters" but I'm not able to enter anything.

Tested with Chrome it works as expected.

Any ideas?

**Comment by szepeviktor** [comment]
Does https://select2.github.io/examples.html work?

**Comment by tmconnect** [comment]
Yes and other select2 fields (e.g. from ACF) working as well.

But it is no conflict with other plugins. Tried it with only Shortcode UI 0.7.2 activated.

**Comment by szepeviktor** [comment]
`define( ‘SELECT2_NOCONFLICT’, true );` ?

**Comment by tmconnect** [comment]
Excuse me, I could have mentioned - yes it's set

**Comment by szepeviktor** [comment]
Possible browser console messages?

**Comment by tmconnect** [comment]
No, nothing. No console messages, no debug messages...

**Comment by szepeviktor** [comment]
Could you inspect the event on the Select2 drop-down?

![select2-event](https://user-images.githubusercontent.com/952007/28239780-fef48b88-6973-11e7-8938-c237693c8262.png)

**Comment by tmconnect** [comment]
Sure, but I see that I got some click events that I don't see on you screenshot. And I see that you use a single-select field and I'm a multiple-select field.

![firefox_screen_001](https://user-images.githubusercontent.com/2598781/28239880-56e221c8-6976-11e7-8517-505cf23c285e.jpg)

I tested the single-select and it worked. But the multiple-select doesn't.

**Comment by nicoladj77** [comment]
I have an issue when I have multiple select2 for Posts select on chrome. @tmconnect is the same issue you are mentioning?
![image](https://user-images.githubusercontent.com/333069/28344957-3a13ff0e-6c27-11e7-811c-59c02acbadd7.png)

**Comment by tmconnect** [comment]
@nicoladj77 No, I have only one select field that is set to select multiple posts and I can't input any text in the search field.

**Comment by scheurta** [comment]
The issue (for me) is that the select2 input field `select2-search__field` has an inline style with `width: 0px;`. The issue disappears when I set a min-width for that input field in the `.edit-shortcode-form`

Example code:

```
add_action( 'admin_head', function() {
	?>
	<style>
		.edit-shortcode-form input.select2-search__field {
			min-width: 50px;
		}
	</style>
	<?php
});
```

I'm using version 0.7.2, I'm not sure if this is fixed in 0.7.3.

**Comment by pySilver** [comment]
@nicoladj77 thats an issue with css: 

```css
// Select2 Fix
.edit-shortcode-form .select2-container {
  z-index: auto;
}
```
